google-cloud-nio/src/test/java/com/google/cloud/storage/contrib/nio/CloudStoragePathTest.java

```diff
@@ -354,7 +342,7 @@ public void testRelativize_providerMismatch2() throws IOException {
       gcs.getPath("/dog").relativize(FileSystems.getDefault().getPath("/etc"));
       Assert.fail();
     } catch (ProviderMismatchException ex) {
-      assertThat(ex.getMessage()).contains("Not a Cloud Storage path: /etc");
+      assertThat(ex.getMessage()).isNotNull();
```

Comment:
In this case, the original assert with contains message was more specific and better